src/components/ConfigMapSecret/ConfigMapSecretForm.tsx

```diff
@@ -531,7 +693,11 @@ export const ConfigMapSecretForm = React.memo(
                                 placeholder={'eg. 0400 or 400'}
                                 error={state.filePermission.error}
                                 onChange={onFilePermissionChange}
-                                disabled={state.cmSecretState === CM_SECRET_STATE.INHERITED || isChartVersion309OrBelow}
+                                disabled={
```

Comment:
Can we have a common const that holds this value?

Reply:
We can but but it has been used only at 2 places so not much beneficial